wiki/Tournament_Drawings/en.md

```diff
@@ -29,22 +37,28 @@ There are two sections to this screen - the main section on the left with the wo
 There are 4 buttons in the control panel, let's go over them:
 
 - **Begin random**
-  - This will start the randomization process, causing the flags on the screen to scroll.
+  - This will start the randomisation process, causing the flags on the screen to scroll.
 - **Stop random**
-  - This will stop the randomization process, causing the scrolling to slow down until eventually stopping and centering on a flag in the middle of the screen.
+  - This will stop the randomisation process, causing the scrolling to slow down until eventually stopping and centring on a flag in the middle of the screen.
 - **Reload**
   - This will reload the `drawings.txt` file.
 - **Reset**
   - This button should rarely be used. It is destructive and will reset the results of the drawings process.
 
-Press the `Begin random` and the `Stop random` buttons. When the scrolling stops on a flag at the centre of the screen, go back to the osu! folder and notice that there is now an additional file created: `drawings_results.txt`.  
-Open the file and observe the format - this is where the results of the drawings process are stored, and should be imported into further tools to assist in managing the tournament, such as Google Spreadsheets.  
+Press the `Begin random` and the `Stop random` buttons. When the scrolling stops on a flag at the centre of the screen, go back to the osu! folder and notice that there is now an additional file created:
+
+```
+drawings_results.txt  
```

Comment:
didn't the previous option work? why put the file name on a separate line?

Reply:
agree https://github.com/ppy/osu-wiki/pull/6485#discussion_r762085383